### ISSUE TITLE
Disconnect/reconnect fails after attaching using `enable_attach`

### ISSUE THREAD

**Issue by karthiknadig** [Microsoft/ptvsd#472]
Disconnecting seems to cause this warning to be written to `stderr`:
```
C:\GIT\ptvsd\ptvsd\wrapper.py:856: UserWarning: timed out waiting for disconnect request
  warnings.warn('timed out waiting for disconnect request')
```

**Comment by karthiknadig** [comment]
See #516 for repro steps.

**Comment by ericsnowcurrently** [comment]
This appears to be fixed with #527.

**Comment by ericsnowcurrently** [comment]
I've verified that it's fixed.  If you notice any issues, feel free to re-open.
